src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/util/SalesforceSourceConstants.java

```diff
@@ -156,4 +162,11 @@ public class SalesforceSourceConstants {
   public static final long GET_BATCH_WAIT_TIME_SECONDS_SERIAL_MODE = 600000;
 
   public static final long MAX_RETRIES_ON_API_FAILURE = 10;
+
+  public static final long DEFAULT_INITIAL_RETRY_DURATION = 5L;
+
+  public static final long DEFULT_MAX_RETRY_DURATION = 80L;
```

Comment:
Add units to the variable name, for example `DEFAULT_INITIAL_RETRY_DURATION_SECONDS`

Reply:
Done